src/editor/components/form/textarea.tsx

```diff
@@ -0,0 +1,18 @@
+interface TextAreaProps {
+  label: string,
+  value: any,
+  placeholder?: string
```

Comment:
These are not needed here

Reply:
why is value not needed? it is part of the input for the TextArea Component.

Reply:
Ah sorry `value` is indeed needed here. In the input however it isn't.